.semaphore/push-images/node.yml

```diff
@@ -57,7 +57,9 @@ blocks:
       jobs:
         - name: Linux arm64
           commands:
-            - if [ -z "${SEMAPHORE_GIT_PR_NUMBER}" ]; then make -C node cd ARCHES=arm64 CONFIRM=true; fi
+            # Don't call `make -C node cd ARCHES=arm64` here because node image-all also builds windows
+            # and fips images that will fail on native arm64 runners. They are built in the previous block.
+            - if [ -z "${SEMAPHORE_GIT_PR_NUMBER}" ]; then make -C node image cd-common ARCH=arm64 CONFIRM=true; fi
```

Comment:
I think we can revert this as well, as windows is done separately

Reply:
`sub-image-fips-amd64` is still a dependency of `image-all` which will fail on arm64 runner (or we current don't support arm64-fips image).

Reply:
Oh I see! I saw the comment mentioning windows and was thrown off by that

Reply:
Comments are updated to mention fips only.